wordpress-objects/Clarkson_Object.php

```diff
@@ -170,6 +170,18 @@ public function get_date( $format = 'U' ) {
 		return date( $format, strtotime( $this->_post->post_date_gmt ) );
 	}
 
+    /**
+     * Get the date localized from the wordpress .mo file.
+     *
+     * @param string $format
+     * @param bool $gmt
+     * @return string
+     */
+    public function get_localized_date( $format = 'U', $gmt = false ) {
```

Comment:
Since we are heading towards more and more projects with multiple languages and since CC is opensource, we should provide localization from within the theme.